### ISSUE TITLE
Collapsible Column Groups the group is not expanded

### ISSUE THREAD

**Issue by hanastasov** [IgniteUI/igniteui-angular#11239]
When the group is the last column in the grid in the grid, and there are no other groups, and the hidden fields are not the last fields in the group, it does not expand the group, and all keyboard navigation fails, and in other cases, the Grid doesn't re-work itself for the space created. I'll add a video

https://www.screencast.com/t/eRNVGkpg

1) Make the group is the last column in the grid and the only one group column
2) Hide some columns from the group (not the last columns)
3) Try to expand the group

## Result  
 it does not expand the group
all keyboard navigation fails, and in other cases, the Grid doesn't re-work itself for the space created

## Expected result  
Group is expanded

## Attachments  
[Attach a sample if available, and screenshots, if applicable.  ](https://www.screencast.com/t/eRNVGkpg)

**Comment by ddincheva** [comment]
This issue has been introduced as a regression with these two bug fixes #10664 and #11509
